clients/spark/core/src/main/scala/io/treeverse/gc/DataLister.scala

```diff
@@ -50,9 +50,14 @@ class ParallelDataLister extends DataLister with Serializable {
   override def listData(configMapper: ConfigMapper, path: Path): DataFrame = {
     import spark.implicits._
     val slices = listPath(configMapper, path)
-    val objectsPath = path.toString // udf require serializable string and not Path
+    var objectsPath = path.toString // udf require serializable string and not Path
```

Comment:
Please keep it immutable (a `val`), by setting it to a conditional expression.

Reply:
Done